webrender/src/platform/unix/font.rs

```diff
@@ -87,17 +90,18 @@ impl FontContext {
     pub fn add_raw_font(&mut self, font_key: &FontKey, bytes: &[u8], index: u32) {
```

Comment:
Raw fonts are passed around as an `Arc<Vec<u8>>`. I think that you could change add_raw_font to take the Arc nstead of a slice and just store the arc to make sure it lives long enough (and that way avoid the copy).